main/guides/getting-started/README.md

```diff
@@ -297,4 +297,17 @@ When the transaction is complete you will notice some IST has been debited from
 
 ![Finished transaction](./assets/new_006.png)
 
-Congratulations! You got your first Agoric dapp up and running!
+Congratulations! You got your first Agoric dapp up and running! Now that you've got your first Agoric dapp up and running, let's focus on some key takeaways from this exercise:
+- Starting an Agoric Chain: You have learned the process to initialize and start a local Agoric blockchain. 
+```sh
+yarn start:docker
+```
+
+- Deploying a Contract: You have learned the steps to deploy a smart contract onto the Agoric platform!
+```sh
+yarn start:contract
```

Comment:
Again, these are not **the** steps to deploy a contract. There are lots of other ways.

I'm still interested in feedback on #906 . If we incorporated that, we'd be in a better position to say that readers learned something about deployment. With what we have now, they do manage to get it done, but I wouldn't say that they learn how it works.

Reply:
Just approved #906, I think it resolves the need for the code one-liners here. I didn't see that one in my notifications though - is that because it was in a draft state? Any way to get draft state notifications?

Reply:
I think I get the same notifications for draft as non-draft, but who knows whether that's cuz I tweaked something.